iphone/cli/commands/_build.js

```diff
@@ -5853,10 +5854,12 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 										// Read the possibly modified file contents
 										const source = r.contents;
 										// Analyze Ti API usage, possibly also minify/transpile
+										// DO NOT TRANSPILE CODE inside SDK's common folder. It's already transpiled!
+										const transpile = from.startsWith(sdkCommonFolder) ? false : this.transpile;
 										const analyzeOptions = {
 											filename: from,
 											minify: this.minifyJS,
```

Comment:
I think we should also toggle off minification for the sdk common dir, while testing this PR it kept choking on the core-js libs

Reply:
ok, turned off minifying those files as well.